gui/src/pages/NewProcess.svelte

```diff
@@ -48,97 +50,98 @@
 </script>
 
 <Layout showBackButton backButtonRoute={workspaceRoute}>
-  <form
-    on:submit|preventDefault={async () => {
-      updateFields();
-      try {
-        const { id } = await workspace.createProcess(name, {
-          directory,
-          environment,
-          program,
-          arguments: args,
-        });
-        setLogVisibility(id, true);
-
-        router.push(workspaceRoute);
-      } catch (ex) {
-        if (!isClientError(ex)) {
-          throw ex;
-        }
-        error = ex;
-      }
-    }}
-  >
-    <div>
-      <label for="name">Name:</label>
-      <Textbox id="name" name="name" bind:value={name} />
-    </div>
-    <div class="edit-as">
-      <span>Edit as:</span>
-      <button
-        class:selected={!structured}
-        on:click|preventDefault={(e) => {
-          if (!structured) {
-            return;
-          }
-          structured = false;
-          updateScript();
-        }}
-      >
-        script
-      </button>
-      <button
-        class:selected={structured}
-        on:click|preventDefault={() => {
-          if (structured) {
-            return;
+  <WithLeftWorkspaceNav {workspaceId} active="Dashboard">
```

Comment:
I'd suggest making this its own component and including it in `Layout` rather than using it as a higher order component that wraps the page content.